### DISCUSSION TITLE
Calling all Tethys Platform Contributors!

### DISCUSSION THREAD

**Discussion by swainn** [tethysplatform/tethys#820] [General]
Tethys Platform Contributors (experienced and aspiring):

We are hard at work on Tethys Platform 4 and we could use some help to get it ready for release. Most of the major breaking changes have been completed and are merged into the `master` branch. However, there are several issues that need to be taken care of before we can release (see: [Milestone Version 4.0](https://github.com/tethysplatform/tethys/milestone/14)).

If you are interested in helping out, please review the issues and post a comment on the ones that you would like to work on. Then submit a pull request to the `master` branch when you are done.

Thank you in advance for your help!

--
Tethys Platform Steering Committee

**Comment by danames** [comment]
I have it on good authority that @shawncrawley has done a lot of cool things with Tethys in recent years and that we should be leaning on him to get his updates integrated. (As per discussion with Ed Clark last week!)
